spikeinterface_gui/tests/iframe/iframe_server.py

```diff
@@ -0,0 +1,121 @@
+import threading
```

Comment:
why having this in test ?

Reply:
this is a test for embedding in a parent app.